README.md

```diff
@@ -107,40 +107,10 @@ $ docker exec -t NAME_OR_ID_OF_YOUR_WALLABAG_CONTAINER /var/www/wallabag/bin/con
 
 ## docker-compose
 
-It's a good way to use [docker-compose](https://docs.docker.com/compose/). Example:
-
-```
-version: '3'
```

Comment:
this line is deprecated in the modern compose file

> The top-level version property is defined by the Compose Specification for backward compatibility. It is only informative and you'll receive a warning message that it is obsolete if used.

https://docs.docker.com/reference/compose-file/version-and-name/#version-top-level-element-obsolete